src/ItemAntivirus.php

```diff
@@ -329,64 +316,53 @@ private static function showForItem(CommonDBTM $asset, $withtemplate = 0)
-                                            __('%1$s = %2$s'),
-                                            $asset->getTypeName(1),
-                                            $asset->getName()
-                                        )
-            );
-
-            $antivirus = new self();
-            foreach ($result as $data) {
-                 $antivirus->getFromDB($data['id']);
-                 echo "<tr class='tab_bg_2'>";
-                 echo "<td>" . $antivirus->getLink() . "</td>";
-                 echo "<td>" . Dropdown::getYesNo($data['is_dynamic']) . "</td>";
-                 echo "<td>";
-                if ($data['manufacturers_id']) {
-                    echo Dropdown::getDropdownName(
-                        'glpi_manufacturers',
-                        $data['manufacturers_id']
-                    ) . "</td>";
-                } else {
-                    echo "</td>";
-                }
-                echo "<td>" . $data['antivirus_version'] . "</td>";
-                echo "<td>" . $data['signature_version'] . "</td>";
-                echo "<td>" . Dropdown::getYesNo($data['is_active']) . "</td>";
-                echo "<td>" . Dropdown::getYesNo($data['is_uptodate']) . "</td>";
-                echo "<td>" . Html::convDate($data['date_expiration']) . "</td>";
-                echo "</tr>";
-                Session::addToNavigateListItems(__CLASS__, $data['id']);
-            }
-            echo $header;
-        } else {
-            echo "<tr class='tab_bg_2'><th colspan='$colspan'>" . __('No item found') . "</th></tr>";
-        }
+        TemplateRenderer::getInstance()->display('components/form/item_antivirus_item.html.twig', [
+            'canedit'               => ($canedit && !(!empty($withtemplate) && ($withtemplate == 2))),
+            'has_antivirus'         => ($result->numrows() != 0),
+            'antivirus_form_url'    => ItemAntivirus::getFormURL() .
+                                        "?itemtype=$itemtype&items_id=$ID&withtemplate=" . $withtemplate,
+        ]);
 
-        echo "</table>";
-        echo "</div>";
+        $antivirus = new self();
+        $entries = [];
+        foreach ($result as $data) {
+            $antivirus->getFromDB($data['id']);
+            $manufacturer = new Manufacturer();
+            $manufacturer->getFromDB($data['manufacturers_id']);
+            $entries[] = [
+                'name'          => $antivirus->getLink(),
+                'is_dynamic'    => Dropdown::getYesNo($data['is_active']),
+                'manufacturers_id' => $manufacturer->getLink(),
+                'antivirus_version' => $data['antivirus_version'],
+                'signature_version' => $data['signature_version'],
+                'is_active'     => Dropdown::getYesNo($data['is_active']),
+                'is_uptodate'   => Dropdown::getYesNo($data['is_uptodate']),
+                'date_expiration' => Html::convDate($data['date_expiration']),
```

Comment:
Date formatting should be done in Twig via the `date` or `datetime` formatter for the datatable template.